ui/components/app/permission-page-container/permission-page-container.component.js

```diff
@@ -103,6 +104,17 @@ export default class PermissionPageContainer extends Component {
 
     if (Object.keys(request.permissions).length > 0) {
       approvePermissionsRequest(request);
+      const searchParams = new URLSearchParams(
+        this.props.history.location.search,
+      );
+      const redirect = searchParams.get('redirect');
+      if (redirect) {
+        window.open(
```

Comment:
It seems that we have been using `global.platform.openTab` in many places, so we can use it here as well to be more consistent.